### DISCUSSION TITLE
Welcome to OctoPrint-DisplayPanel Discussions!

### DISCUSSION THREAD

**Discussion by sethvoltz** [sethvoltz/OctoPrint-DisplayPanel#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and build photos
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GilH94** [comment]
Bonjour, Hi

---- French ---
Bonjour,

Je trouve qu'il s'agit d'un projet passionnant.
J'ai créer la télécommande et son câblage (j'ai tester le bon fonctionnement sur un esp8266). Lorsque je fait un "$ sudo i2cdetect -y 1" , j'ai bien 30: -- -- -- -- -- -- -- -- -- -- -- -- 3c.Mais l'écran reste déseperement éteint une fois connecter sur les bon port GPIO avec i2c activée.
Version python --version = Python 3.7.3
Et https://github.com/sethvoltz/OctoPrint-DisplayPanel/archive/main.zip installé dans le plug-in manager.
 Je n'ai plus d'idée  ou chercher le défaut ! Ne connaissant pas suffisamment l'anglais je ne trouve pas de solution dans les forum. Si a tout hasard vous pouvez m'aider je vous en serais reconnaissant. Merci

--- Google Trad --
Hello,

I find this to be an exciting project.
I created the remote control and its wiring (I tested the correct operation on an esp8266). When I do a "$ sudo i2cdetect -y 1", I have 30: - - - - - - - - - - - - 3c. But the screen remains hopelessly off once connect to the correct GPIO port with i2c enabled.
Python version --version =Python 3.7.3 
And https://github.com/sethvoltz/OctoPrint-DisplayPanel/archive/main.zip installed in the plug-in manager.
 I have no idea where to look for the fault! Not knowing enough English, I cannot find a solution in the forums. If on any chance you can help me I would appreciate it. Thank you

**Reply by sethvoltz** [reply]
Please confirm that your Octoprint version is also on Python 3. The Octoprint version is often different than the Raspberry Pi system version. If you scroll to the bottom of an Octoprint page, in the lower left there should be something like this:
![image](https://user-images.githubusercontent.com/39506/109433549-bf4dd780-79c5-11eb-95fe-5ac3220bc435.png)

Additionally, if you are using a Raspbberry Pi 4 with a recent install of OctoPi, please try again after version 2.1.0 is available (I released it today) as it includes a fix for a dependency that is not automatically installed on Pi 4 installs.

Thanks you,
Seth
